sympy/geometry/ellipse.py

```diff
@@ -694,6 +694,11 @@ def is_tangent(self, o):
         >>> l1 = Line(p1, p2)
         >>> e1.is_tangent(l1)
         True
+        
+        References
+        ==========
+
+        [1] http://jwilson.coe.uga.edu/emt669/student.folders/banker.teresa/unitstdy/unitstdy.html
```

Comment:
This `student.folders` link is likely to disappear in a short while. Would it be possible to find a more stable reference?

Reply:
@ylemkimon, I tried to find a more stable reference.<br> I mentioned earlier, I did not find a definition for tangent ray/segment (other than this ). <br>In most of the places tangent is defined as a tangent line, but I found following definition at (https://en.wikipedia.org/wiki/Tangent_(disambiguation))[https://en.wikipedia.org/wiki/Tangent_(disambiguation)]<br><br>More generally, in geometry, two curves are said to be tangent when they intersect at a given point and have the same direction at that point